src/budget/budgetproposal.h

```diff
@@ -71,6 +70,7 @@ class CBudgetProposal
 
     bool IsEstablished() const;
     bool IsPassing(int nBlockStartBudget, int nBlockEndBudget, int mnCount) const;
+    bool IsExpired(int nCurrentHeight);
```

Comment:
(467a8e4052fe961a430be2190a0c02b6e2cf320d)

The position of `IsExpired` in the header file was not random. It was listed along with the other functions used to check/update the validity (as per comment above them).

Reply:
k, i moved it because needed public access to it but yeah.. will add a const version of the function that does not modifies any data of the proposal.